extension/src/background/helpers/dataStorage.ts

```diff
@@ -34,3 +34,11 @@ export const migrateLocalStorageToBrowserStorage = async () => {
 
   await dataStorage.setItem(storage);
 };
+
+// TODO - temporary wrapper around localStorage until we replace
+// localStorage all together
+export const freighterLocalStorage = {
+  getItem: (keyId: string) => localStorage.getItem(keyId),
```

Comment:
Kind of annoying, but can you make these methods (getItem, setItem, clear) async? The browser.storage API is async, so at some point this will need to be async. It'd be nice if we started calling `await backgroundDataStorage` now so the call signature wouldn't have to change in the future